### PR TITLE
Thrown items only make a sound if they have throwforce

### PR DESCRIPTION
**Pull request by Rohesie**
It's kind of odd hearing the thrown sound of small, harmless objects.
This way the sound will only happen when they are able to hurt someone on throw.
Might also be a good reminder on what not to be throwing carelessly around.

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -202,7 +202,8 @@
 		visible_message("<span class='danger'>[src] has thrown [thrown_thing].</span>")
 		log_message("has thrown [thrown_thing]", LOG_ATTACK)
 		do_attack_animation(target, no_effect = 1)
-		playsound(loc, 'sound/weapons/punchmiss.ogg', 50, 1, -1)
+		if(throwforce)
+			playsound(loc, 'sound/weapons/punchmiss.ogg', 50, TRUE, -1)
```
Comment (Afya212) [inline code/modules/mob/living/carbon/carbon.dm:206]: Don't forget to mark it as a Skyrat change if it isn't in modular. 
```suggestion
		if(throwforce) //Skyrat change
			playsound(loc, 'sound/weapons/punchmiss.ogg', 50, TRUE, -1) //Skyrat change
```

**Comment by BobJoga** [comment]
Ehhh, this might create "problems" with some dumb items like plushies and bike horns.

**Comment by BobJoga** [comment]
nevermind i am an idiot

File: code/modules/mob/living/carbon/carbon.dm (hunk 1 above)
Comment (Rohesie) [reply code/modules/mob/living/carbon/carbon.dm:207]: Added tags on begin and end, which should be hopefully a bit more visible.

**Pull request merged**